### PR TITLE
Add collection, crosshair, height and width props to timeseries chart

### PR DESCRIPTION
**Pull request by follan**

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-202.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-202.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-202.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesChart/TimeseriesChart.tsx [hunk 1]
```diff
@@ -18,21 +18,25 @@ import { Spin } from 'antd';
 import { decimalTickFormatter, getColor } from '../../utils';
 
 export type TimeseriesChartProps = {
-  panelHeight: number;
+  containerStyle: React.CSSProperties;
```
Comment (mrtraser) [inline src/components/TimeseriesChart/TimeseriesChart.tsx]: Could we have a prop `styles` like [TenantSelector - Custom Styling](https://github.com/cognitedata/gearbox.js/pull/187/files#diff-14f16035ee023b72b8993bcb65372f41R43) with field `container`?

Comment (follan) [reply src/components/TimeseriesChart/TimeseriesChart.tsx]: Like this?

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-202.gearboxjs.preview.cogniteapp.com

**Pull request merged**